boards/frdm-common/dist/openocd-dap-klx.cfg

```diff
@@ -0,0 +1,25 @@
+#
```

Comment:
Just wondering, could this be adapted to #7686?

Reply:
Yes, will update

Reply:
addressed